src/Controller/Component/AuthComponent.php

```diff
@@ -170,21 +172,11 @@ class AuthComponent extends Component
     protected $_authorizeObjects = [];
 
     /**
-     * The session key name where the record of the current user is stored. Default
-     * key is "Auth.User". If you are using only stateless authenticators set this
-     * to false to ensure session is not started.
+     * Storage object.
      *
-     * @var string
-     */
-    public $sessionKey = 'Auth.User';
```

Comment:
If we want everything to be totally BC then yeah it probably should. As an end developer I would expect that the code I had in 3.0 would continue to work the same way in 3.1.